pkg/util/detector/detector.go

```diff
@@ -935,7 +934,7 @@ func (d *ResourceDetector) OnResourceBindingDelete(obj interface{}) {
 	}
 }
 
-// ReconcileClusterResourceBinding handles ResourceBinding object changes.
+// ReconcileClusterResourceBinding handles ClusterResourceBinding object changes.
 // For each ClusterResourceBinding changes, we will try to calculate the summary status and update to original object
 // that the ClusterResourceBinding refer to.
 func (d *ResourceDetector) ReconcileClusterResourceBinding(key util.QueueKey) error {
```

Comment:
Why not remove the method completely?  Furthermore, the `clusterBindingReconcileWorker` should be cleaned up too.